<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoomsDock</class>
 <widget class="QDockWidget" name="RoomsDock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="features">
   <set>QDockWidget::DockWidgetFloatable|QDockWidget::DockWidgetMovable|QDockWidget::DockWidgetVerticalTitleBar</set>
  </property>
  <property name="windowTitle">
   <string>Chat rooms</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="margin">
     <number>2</number>
    </property>
    <item>
     <widget class="QTableView" name="tblRooms"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>3</number>
      </property>
      <item>
       <widget class="QLineEdit" name="txtRoomName"/>
      </item>
      <item>
       <widget class="QPushButton" name="butJoinRoom">
        <property name="toolTip">
         <string>Join the room</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../ressources.qrc">
          <normaloff>:/icons/ressources/join_chat_room.png</normaloff>:/icons/ressources/join_chat_room.png</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
